fermipy/defaults.py

```diff
@@ -101,7 +101,6 @@ def make_default_dict(d):
     'resample': (True, '', bool),
     'srcmap': (None, '', str),
     'bexpmap': (None, '', str),
-    'wmap': (None, 'Likelihood weights map.', str),
```

Comment:
Hi Matt-

Im really sorry if Im mixing things up- I actually didnt change that code
in defaults.py - I only added the defaults for the LC functionality. I
didnt take anything out (that I know of)

To commit and push Im using

git commit -m ''
and git push origin lightcurve-dev

I thought this would only push changed to the lightcurve-dev branch. How do
I avoid rolling over recent changes in the master?

Andy




On Wed, Aug 31, 2016 at 6:32 PM, Matthew Wood <notifications@github.com>
wrote:

> In fermipy/defaults.py
> <https://github.com/fermiPy/fermipy/pull/80#discussion_r77102048>:
>
> > @@ -101,7 +101,6 @@ def make_default_dict(d):
> >      'resample': (True, '', bool),
> >      'srcmap': (None, '', str),
> >      'bexpmap': (None, '', str),
> > -    'wmap': (None, 'Likelihood weights map.', str),
>
> It looks like this commit is reverting recent changes in gtanalysis and
> defaults. We'll need to fix this before this can be merged.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fermiPy/fermipy/pull/80/files/439d4ebc8d1496b32bbe6993e70bf4149628eba7..ec075b26ee7244283d6c6c4e634325ad91329f2a#r77102048>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJllvUVsmzro3AT8QsPCF7ZJIyi4tVf0ks5qlisagaJpZM4JtMuE>
> .
>



-- 
Dr. Andrew W. Smith
Research Associate Professor
Department of Physics
University of Maryland, College Park
1117 John S. Toll Building #082
College Park, MD 20742